### PR TITLE
feat: migration cpms-notifications to mot-cpms-notifications

### PR DESCRIPTION
**Pull request by caoimheb**
## Description

Migrate cpms-notifications to mot-cpms-notifications

Related issue: [BL-14414](https://dvsa.atlassian.net/browse/BL-14414)

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code?
- [ ] Have you have added tests that prove the fix or feature is effective and working?
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "dvsa/mot-cpms-notifications",
+    "type": "library",
+    "license": "proprietary",
+    "repositories": [
```
Comment (caoimheb) [inline composer.json]: TODO Remove - after `cpms-queues` is Done

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            "type": "composer",
+            "url": "https://packages.mot.dvsacloud.uk/"
+        }
+    ],
+    "require": {
+        "php": "^7.3 || ^8.0",
+        "cpms/cpms-queues": "^1.13.0",
```
Comment (caoimheb) [inline composer.json]: TODO - after `cpms-queues` ticket is Done

```suggestion
        "dvsa/mot-cpms-queues": "^1.0.0",
```

File: phpcs.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="mot-logger" xsi:noNamespaceSchemaLocation="./vendor-bin/php-codesniffer/vendor/squizlabs/php_codesniffer/phpcs.xsd">
```
Comment (jennifer-choo) [inline phpcs.xml]: The name needs updating on this
```suggestion
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="mot-cpms-notifications" xsi:noNamespaceSchemaLocation="./vendor-bin/php-codesniffer/vendor/squizlabs/php_codesniffer/phpcs.xsd">
```

**Review by DariuszJerzewski — APPROVED** [review]

File: phpcs.xml (hunk 3 above)
Comment (faggc) [reply phpcs.xml]: resolved?

**Review by faggc — APPROVED** [review]

**Review by jennifer-choo — APPROVED** [review]

**Pull request merged**